provider/baked/src/export.rs

```diff
@@ -132,6 +128,9 @@ fn maybe_msrv() -> TokenStream {
 #[non_exhaustive]
 #[derive(Debug, Clone, Copy)]
 pub struct Options {
+    /// By default, baked providers perform fallback internally. This field can be used to
+    /// disable this behavior.
+    pub with_fallback: bool,
```

Comment:
Suggestion:

```suggestion
    pub use_internal_fallback: bool,
```

based on the existing option `use_separate_crates`